android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -193,6 +194,14 @@ open class Node<V : RibView>(
     }
 
     open fun onDetach() {
+        if (isViewAttached) {
```

Comment:
can we cover this with a test?

Reply:
It is covered in `ConfigurationFeatureTest`, look for test cases containing "inOrder".

Reply:
I mean the logic that automatically detaches the view and reports a non-fatal error if the view is not detached before `Node.onDetach()`

Reply:
Done.